packages/hurl/src/cli/options/mod.rs

```diff
@@ -415,6 +419,7 @@ impl CliOptions {
         let follow_location = self.follow_location;
         let follow_location_trusted = self.follow_location_trusted;
         let from_entry = self.from_entry;
+        let headers = self.headers.clone();
```

Comment:
This clone is unnessary, as we only need a reference to pass to `RunnerOptionsBuilder::new().headers()`
We could just done:

```rust
let headers = &self.headers;
```
And

```rust
        RunnerOptionsBuilder::new()
            .aws_sigv4(aws_sigv4)
            ...
            .headers(headers)
```